apps/designer/remix.config.js

```diff
@@ -17,4 +17,7 @@ module.exports = {
   // assetsBuildDirectory: "public/build",
   serverBuildPath: "api/index.js",
   // publicPath: "/build/",
+  watchPaths: () => {
+    return ["../../packages/react-sdk/lib/**/*"];
```

Comment:
the problem is that its not only react-sdk, basically we need to watch all of them or otherwise we will have to add each time we add a new package and we will have many more packages there

Reply:
Okay, will change it to watch the whole packages folder.